src/mol-plugin/util/headless-screenshot.ts

```diff
@@ -8,33 +8,32 @@
  * @author Adam Midlik <midlik@gmail.com>
  */
 
-import * as fs from 'fs';
```

Comment:
you can import fs and path normally here since the file is only to be used with nodejs